dlib/dnn/layers_abstract.h

```diff
@@ -939,6 +939,30 @@ namespace dlib
                 - #get_bias_weight_decay_multiplier() == val
         !*/
 
+        void disable_relu(
+        );
+        /*!
+            ensures
+                - relu_is_disabled() returns true
+                - a relu activation will be applied after convolution when calling forward()
+        !*/
+
+        void enable_relu(
+        );
+        /*!
+            ensures
+                - relu_is_disabled() returns false
+                - no relu activation will be applied after convolution when calling forward()
+        !*/
```

Comment:
```suggestion
        void disable_relu(
        );
        /*!
            ensures
                - relu_is_disabled() returns true
        !*/

        void enable_relu(
        );
        /*!
            ensures
                - relu_is_disabled() returns false
        !*/
```
These are backwards.  Don't need to be here anyway, since `relu_is_disabled()` defines it clearly and this only duplicates the docs so it's better if they aren't there in any case :)